lib/utilities/reportHandler.js

```diff
@@ -73,6 +74,56 @@ function getSystemStats() {
     };
 }
 
+function getCRRStats(log, cb) {
+    log.debug('getting CRR stats', { method: 'getCRRStats' });
+    // TODO: Reuse metrics code from Backbeat by moving it to Arsenal instead of
+    // making an HTTP request to the Backbeat metrics route.
+    const params = { url: 'http://localhost:8900/_/metrics/crr/all' };
+    return request.get(params, (err, res) => {
+        if (err) {
+            log.error('failed to get CRR stats', {
+                method: 'getCRRStats',
+                error: err,
+            });
+            return cb(err);
+        }
+        let body;
+        try {
+            body = JSON.parse(res.body);
+        } catch (parseErr) {
+            log.error('could not parse backbeat response', {
+                method: 'getCRRStats',
+                error: parseErr,
+            });
+            return cb(errors.InternalError
+                .customizeDescription('could not parse backbeat response'));
+        }
+        const { completions, backlog, throughput } = body;
+        if (!completions || !backlog || !throughput) {
+            log.error('could not get metrics from backbeat', {
+                method: 'getCRRStats',
+            });
+            return cb(errors.InternalError
+                .customizeDescription('could not get metrics from backbeat'));
+        }
+        const stats = {
+            completions: {
+                count: parseFloat(completions.results.count),
```

Comment:
The swagger definition expects a `int64` instead of a float

Reply:
I think the approximation is fine, the volume of data that zenko's meant to handle makes the rounding errors not matter much :)